pkg/doc/did/serialize_default.go

```diff
@@ -7,6 +7,9 @@ SPDX-License-Identifier: Apache-2.0
 
 package did
 
+// nolint:gochecknoglobals
+var doACAPYInterop = false
```

Comment:
This should be a constant not a variable.

Reply:
fixed